batch_request_test.go

```diff
@@ -171,7 +172,14 @@ func TestHandlesHTTPError(t *testing.T) {
 	require.NoError(t, err)
 
 	_, err = batch.Send(context.Background(), reqAdapter)
-	assert.Equal(t, err.Error(), "content is empty")
+
```

Comment:
Error message changed with  [this modification ](https://github.com/microsoft/kiota-http-go/pull/152) on http-go.
Updated the error to check on type and message